Example/ExampleAPITests/Endpoint/FileUploadEndpointTests.swift

```diff
@@ -15,7 +15,7 @@ final class FileUploadEndpointTests: XCTestCase {
         let fileURL = URL(string: "path/to/file")!
         let endpoint = FileUploadEndpoint(fileURL: fileURL)
         
-        let (request, body) = try endpoint.makeRequest()
+        let (request, body) = try! endpoint.makeRequest().get()
```

Comment:
Do we need force unwrap, I see the method marked as `throws`.

Reply:
`makeRequest` is no longer throws. This `try!` is for `get()` method of `Result`

Reply:
You don’t need to force unwrap here. Just remove ”!” symbol.